client/src/game/ui/hypothetical.ts

```diff
@@ -46,6 +46,8 @@ export const start = () => {
     replay.toggleSharedTurns();
   }
 
+  globals.elements.toggleRevealedButton?.setEnabled(true);
```

Comment:
should be a ! instead of a ? i think

Reply:
we want to error out if the button doesn't exist for whatever reason?

Reply:
i guess not

Reply:
if we want to error out, why does the type allow null/undefined?

Reply:
because its always supposed to exist but you can't instantiate it in the constructor because typescript is dumb